File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.js
function solution(s, skip, index) {
    let alp="abcdefghijklmnopqrstuvwxyz";
    let result="";
    let skl=skip.length;
    let sl=s.length;
    for(let i=0;i<skl;i++){
        alp=alp.replace(skip[i],"")
    }
    let al=alp.length;
    for(let i=0;i<sl;i++){
        result+=alp[(alp.indexOf(s[i])+index)%al];
    }
    return result;
}

File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.py
def solution(s, skip, index):
    alp = 'abcdefghijklmnopqrstuvwxyz'
    result=""
    l=len(alp)
    for char in s:
        idx=alp.index(char)+1
        count=0
        while count!=index:
            # if idx>l:
                # break
            if alp[idx%l] not in skip:
                count+=1
            idx+=1
        result+=alp[idx%l-1]
    return result